### PR TITLE
feat: homepage, quiz-selection, rules

### PR DESCRIPTION
**Pull request by ParthParolekar**
Added homepage and quiz selection page

### CODE REVIEW

**Comment by ParthParolekar** [comment]
image

image

image

**Review by VLeads — APPROVED** [review]
Nice UI and code is good too

**Review by partha8 — APPROVED** [review]
looks good

File: pages/category/category.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.0/css/all.min.css"
+      integrity="sha512-BnbUDfEUfV0Slx6TunuB042k9tuKe3xrD6q4mg5Ed72LTgzDIcLPxg6yI2gcMFRyomt+yJJxE+zJwNmxki6/RA=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>Chooes Quiz</title>
+  </head>
```
Comment (anik31) [inline pages/category/category.html]: typo

**Comment by anik31** [comment]
The select another quiz button is not readable. It may have low color contrast, you can check it out.

**Comment by ParthParolekar** [comment]
> The select another quiz button is not readable. It may have low color contrast, you can check it out.

Changed the button and fixed the typo! Thank you for pointing it out!

**Review by Developers-codz — APPROVED** [review]
Looks Good

**Comment by ParthParolekar** [comment]
Thank you @anik31 @Developers-codz @partha8  and @VLeads  for the reviews. Merging now

**Pull request merged**